tests/test_config_hub.py

```diff
@@ -11,6 +11,7 @@
     ("litgpt/pretrain.py", "pretrain/debug.yaml"),
     ("litgpt/pretrain.py", "pretrain/tinyllama.yaml"),
     ("litgpt/pretrain.py", "pretrain/tinystories.yaml"),
+    ("litgpt/pretrain.py", "https://raw.githubusercontent.com/Lightning-AI/litgpt/wip/config_hub/pretrain/tinystories.yaml"),
```

Comment:
This parametrization could become a problem if we make breaking changes so that old configs cannot be loaded anymore. At least this will warn us about it